packit_service/models.py

```diff
@@ -202,7 +202,7 @@ def get_by_id(cls, id_: int) -> Optional["CoprBuild"]:
     @classmethod
     def get_all(cls) -> Optional[Iterable["CoprBuild"]]:
         with get_sa_session() as session:
-            return session.query(CoprBuild).all()
+            return session.query(CoprBuild).order_by(desc(CoprBuild.id)).all()
```

Comment:
ordering by ID is yucky, but it's better than what we have now and since not all the builds have datetime attribs in prod, I'd say this is good enough